### DISCUSSION TITLE
Extracting a twitter (X) list ... into a bsky list ?

### DISCUSSION THREAD

**Discussion by solertoyo** [kawamataryo/sky-follower-bridge#121] [Ideas]
I just discovered extracting lists in twitter to follow accounts in BSky is possible ! Great
I simply wonder if we could directly generate (or add) theses accounts found on twitter into a list (and not following them), & if not, could it be a next step ?
--> Thanks a lot for this more than useful work anyway...

**Comment by kawamataryo** [comment]
Thank you for your question! I'm actually in the process of creating that feature right now. It will be released soon, so please look forward to it!
